src/customize-pop/accessibility-settings.md

```diff
@@ -0,0 +1,17 @@
+# Accessibility Settings
+
+Options to configure accessibility settings are located in `Settings` ➞ `Accessibility`. Accessibility features provide visual and audio enhancements to highlight system actions for greater usability. This section briefly describes each setting, but more information can be found on [GNOME's help page](https://help.gnome.org/users/gnome-help/stable/a11y.html).
+
+![Accessibility Settings](/images/accessibility-settings/accessibility-settings.png)
+
+## Always Show Accessibility Menu
+
+When the `Always Show Accessibility Menu` is enabled, accessibility options can be toggled using an icon in the upper right corner.
+
+![Menu in Tray](/images/accessibility-settings/menu-in-tray.png)
+
+**Click image to enlarge**
```

Comment:
Hmm, how much is this really adding since we have the close-up view just above? Would it be simpler to show the option first, then the close-up?

Reply:
Changed to close-up of menu option followed by close-up of "always show" dropdown, and removed the html to enlarge the original image.